src/Bosnadev/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -28,47 +35,30 @@ protected function getDoctrineColumnType($type)
     }
 
     /**
-     * Create the column definition for a character type.
+     * Handle dynamic method calls.
      *
-     * @param Fluent $column
-     * @return string
+     * @param  string $method
+     * @param  array $parameters
+     * @return mixed
      */
-    protected function typeCharacter(Fluent $column)
+    public function __call($method, $parameters)
     {
-        return "character({$column->length})";
-    }
+        if(substr($method, 0, 4) === 'type') {
+            $type = substr($method, 4, strlen($method) - 4);
```

Comment:
@khalilst it should also use `strtolower` to ensure types work no matter how many capital letters are used.